test/ibmq/test_ibmq_integration.py

```diff
@@ -143,3 +148,84 @@ def test_execute_two_remote(self, provider):
         job = execute([qc, qc_extra], backend, seed_transpiler=self.seed)
         results = job.result()
         self.assertIsInstance(results, Result)
+
+    def test_ensure_backend_jobs_signature(self):
```

Comment:
`test_ibmq_integration` seems to be an odd place to put these tests, since this test file is for "integration test" (i.e. "compile and run" jobs). Maybe you can move the backend related tests to `test_ibmq_backend` (under a different class) and `test_ibmq_managed_job_signature` to `test_ibmq_jobmanager` ?